### DISCUSSION TITLE
Student Developer Pack

### DISCUSSION THREAD

**Discussion by JinxX404** [community/community#68830] [Other Feature Feedback, Questions, & Ideas]
How do I add edu account to my personal account?
if i have personal account  "xx@personal.com"  and educational account "yy@edu.com" as a separeted accounts, both of which I signed up for individually.
and I need to add "yy@edu.com" to my personal account in order to access the GitHub Student Developer Pack , I encounter an error message stating, "email is already in use."
how can I solve this issue?

**Comment by vishalkrishna8** [comment]
Hii @MoatazXI 

First log in to your personal GitHub account and navigate to your account settings. In the settings, select "Emails." Here, you can add your educational email and verify it through a confirmation email sent by GitHub. You can also choose to make the educational email your primary address if desired. Once your educational email is linked and verified, you'll be able to apply for the GitHub Student Developer Pack using your educational status. If you encounter any issues like an "email is already in use" error, ensure that your educational email isn't associated with other GitHub accounts

**Reply by JinxX404** [reply]
my educational email not associated with other account as "its edu account"
but my educational email is a separted account i already sign up with it.
what i need is merge this edu account to my personal account to get student developer pack.
in this case if i deleted my edu account, then can i use it in my personal account?

**Reply by vishalkrishna8** [reply]
As per my knowledge GitHub doesn't provide a direct merging option. Instead, you can add your educational email to your personal account, verify it, and apply for the Student Developer Pack.

**Reply by JinxX404** [reply]
I couldn't solve it because in verification step i get this message "email already in use"

**Reply by vishalkrishna8** [reply]
It means your educational email associated with other GitHub accounts
